### ISSUE TITLE
twisted.application.strports should be ported to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8011]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8011|
|Type|enhancement|
|Created|2015-08-30 05:54:09Z|
|Branch|https://github.com/twisted/twisted/tree/strports-py3-8011|

Required by [#8008](https://github.com/twisted/twisted/issues/8008)#ticket

<details><summary>Searchable metadata</summary>

```
trac-id__8011 8011
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_strports_py3_8011 branches/strports-py3-8011
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1440914049453369 1440914049453369
changetime__1441012651678674 1441012651678674
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [45576]) Branching to strports-py3-8011.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Builds are looking good, please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Changes look good. Please merge. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45581]) Merge strports-py3-8011: Port twisted.application.strports to Python 3

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8011](https://github.com/twisted/twisted/issues/8011)
